### DISCUSSION TITLE
helm 安装2.2.2 Milvus components 启动不起来

### DISCUSSION THREAD

**Discussion by altqxd** [milvus-io/milvus#21901] [Q&A and General discussion]
![123](https://user-images.githubusercontent.com/119843642/215938585-388b43f9-b05c-4b53-a852-f63904b19157.jpg)
![234](https://user-images.githubusercontent.com/119843642/215938597-d0179e8a-6215-4cc0-8a8a-56ee1f62f493.jpg)
![345](https://user-images.githubusercontent.com/119843642/215938647-b420410f-720f-47b7-9c73-8dba1e0b06d9.jpg)
components 启动不了

**Comment by LoveEachDay** [comment]
@altqxd Could you check the `cpuinfo` of the worker node use the following command:
```
cat /proc/cpuinfo
```
And check the `flags` output, try to find if there are any tags like: `sse4_2`, `avx` , `avx2` , `avx512`.

**Reply by altqxd** [reply]
> @altqxd Could you check the `cpuinfo` of the worker node use the following command:
> 
> ```
> cat /proc/cpuinfo
> ```
> 
> And check the `flags` output, try to find if there are any tags like: `sse4_2`, `avx` , `avx2` , `avx512`.

flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm

没有这种字段

**Reply by altqxd** [reply]
是因为这个原因吗

**Reply by LoveEachDay** [reply]
Yes, milvus requires the above flags enabled. At least one of sse4_2, avx , avx2 , avx512.

**Comment by yhmo** [comment]
@altqxd 先缩小范围排查，试着用docker-compose起一个standalone的milvus看一下是什么表现

**Reply by ruixiu** [reply]
我之前使用helm在Kubesphere上部署，使用pulsar就不能用，换成kafka就正常
